api/v3/openapi.yaml

```diff
@@ -1272,9 +1272,11 @@ paths:
                     record produced. This API can be used in streaming mode by setting "Transfer-Encoding:
                     chunked" header. For as long as the connection is kept open, the server will
                     keep accepting records. For each record sent to the server, the server will
-                    asynchronously send back a delivery report, in the same order. Records are
-                    streamed to and from the server as Concatenated JSON. Errors are reported per
-                    record. The HTTP status code will be HTTP 200 OK as long as the connection is successfully established.
+                    asynchronously send back a delivery report, in the same order. The delivery reports 
```

Comment:
nit: For me (NB: a non-native speaker), this reads slightly better if the sentences here are joined:
_For each record sent to the server, the server will asynchronously send back a delivery report, in the same order, each with their own error_code._ and if _Records are streamed to and from..._ is put before that sentence and just after _...the server will keep accepting records._.